src/oauth-tooling.ts

```diff
@@ -142,13 +142,13 @@ function buildRequestAccessTokenUrl(accessTokenEndpoint: string, queryParams?: O
 
 /**
  * Makes a request to the `tokenInfoUrl` to validate the given `accessToken`.
- * Resolves with an object containing access token information in case of success.
+ * Resolves with an object containing token validation information in case of success.
```

Comment:
Resolves with an token in case of success.

Reply:
👍